test/local-time.js

```diff
@@ -66,4 +66,12 @@ suite('local-time', function() {
     }
     assert.include(['1969', '1970'], root.children[0].textContent)
   })
+
+  ;('Intl' in window ? test : test.skip)('displays time zone name', function() {
+    var time = document.createElement('local-time')
+    time.setAttribute('datetime', '1970-01-01T00:00:00.000Z')
+    time.setAttribute('minute', '2-digit')
+    time.setAttribute('timeZoneName', 'short')
```

Comment:
@dgraham made a point about keeping the attribute names matching the properties in DateTimeFormat: https://github.com/github/time-elements/pull/65#r73983580

I could go either way.